runtime/lua/vim/lsp/util.lua

```diff
@@ -2140,7 +2140,9 @@ end
 ---@param settings table language server settings
 ---@param section  string indicating the field of the settings table
 ---@return table|string|vim.NIL The value of settings accessed via section. `vim.NIL` if not found.
+---@deprecated
 function M.lookup_section(settings, section)
+  vim.deprecate('vim.lsp.util.lookup_section()', 'vim.tbl_get()', '0.11')
```

Comment:
Any reason this deprecation needs to be fast-tracked instead of deprecating in `0.12`?

The other fast-tracked deprecations had explicit justifications: https://github.com/neovim/neovim/pull/26742#issuecomment-1870032292

Reply:
I see. I don't see any particular reason, but was simply following other functions removed from `vim.lsp.util`. We should follow the standard deprecation strategy, let me fix it to 0.12.

Reply:
`vim.deprecate` no longer gives a warning on the first version bump so we can add it immediately.

Reply:
> `vim.deprecate` no longer gives a warning on the first version bump so we can add it immediately. #26735

Understood, my point was about using `'0.12'` instead of `'0.11'` as the final argument, not about removing the call entirely.

Reply:
Updated (24c1c22).